scripts/list.sh

```diff
@@ -11,6 +11,7 @@ DOCKER_VERSIONS=(
   1.8.3
   1.9.1
   1.10.1
```

Comment:
This should be bumped to 1.10.3, this file contains the latest point release of each major version